src/test/java/com/wikia/webdriver/common/core/drivers/browsers/EdgeBrowser.java

```diff
@@ -0,0 +1,29 @@
+package com.wikia.webdriver.common.core.drivers.browsers;
+
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.drivers.BrowserAbstract;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.edge.EdgeOptions;
+
+public class EdgeBrowser extends BrowserAbstract {
+
+  private EdgeOptions edgeOptions = new EdgeOptions();
+
+  @Override
+  public void setOptions() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1186)